client/src/Movies/Movie.js

```diff
@@ -18,7 +19,7 @@ const Movie = (props) => {
           console.error(error);
         });
 
-  },[]);
+  },[props.movies, movie.id, props.match.params.id]);
```

Comment:
I am not a hooks expert so I am curious: why did you use three things in this dependency array?